src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -0,0 +1,201 @@
+  }
+
+  &:hover,
+  &.pf-m-hover {
+    &::before {
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--hover--BorderBottomColor);
+    }
+  }
+
+  &:active,
+  &.pf-m-active {
+    &::before {
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--active--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-options-menu__toggle--active--BorderBottomWidth);
+    }
+  }
+
+  .pf-c-options-menu.pf-m-expanded > & {
+    &::before {
+      --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-c-options-menu__toggle--expanded--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-options-menu__toggle--expanded--BorderBottomWidth);
+    }
+  }
+
+  &.pf-m-plain {
+    color: var(--pf-c-options-menu__toggle--m-plain--Color);
+
+    &::before {
+      border-color: var(--pf-c-options-menu__toggle--m-plain--BorderColor);
+    }
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    .pf-c-options-menu.pf-m-expanded > & {
+      --pf-c-options-menu__toggle--m-plain--Color: var(--pf-c-options-menu__toggle--m-plain--hover--Color);
+
+      &::before {
+        border-color: var(--pf-c-options-menu__toggle--m-plain--BorderColor);
+      }
+    }
+  }
+}
+
+.pf-c-options-menu__toggle-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+.pf-c-options-menu__toggle-icon {
+  margin-left: var(--pf-c-options-menu__toggle-icon--MarginLeft);
+
+  .pf-m-top.pf-m-expanded > & {
+    transform: rotate(-180deg);
+  }
+}
+
+.pf-c-options-menu__menu {
+  position: absolute;
+  top: var(--pf-c-options-menu__menu--Top);
+  z-index: var(--pf-c-options-menu__menu--ZIndex);
+  min-width: 100%;
+  padding-top: var(--pf-c-options-menu__menu--PaddingTop);
+  padding-bottom: var(--pf-c-options-menu__menu--PaddingBottom);
+  background-color: var(--pf-c-options-menu__menu--BackgroundColor);
+  background-clip: padding-box;
+  border: var(--pf-c-options-menu__menu--BorderWidth) solid transparent; // this is for high contrast mode in windows
+  box-shadow: var(--pf-c-options-menu__menu--BoxShadow);
+
+  &.pf-m-align-right {
+    right: 0;
+  }
+
+  .pf-c-options-menu.pf-m-top & {
+    --pf-c-options-menu__menu--Top: var(--pf-c-options-menu--m-top__menu--Top);
+
+    transform: var(--pf-c-options-menu--m-top__menu--Transform);
+  }
+}
+
+.pf-c-options-menu__menu-item {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: var(--pf-c-options-menu__menu-item--PaddingTop) var(--pf-c-options-menu__menu-item--PaddingRight) var(--pf-c-options-menu__menu-item--PaddingBottom) var(--pf-c-options-menu__menu-item--PaddingLeft);
+  white-space: nowrap;
+  border: none;
+
+  &:hover,
+  &.pf-m-hover,
+  &:focus,
+  &.pf-m-focus {
+    --pf-c-options-menu__menu-item--Color: var(--pf-c-options-menu__menu-item--Color);
```

Comment:
should this just be `color: var(--pf-c-options-menu__menu-item--Color)` ? I don't see `--pf-c-options-menu__menu-item--Color` being defined before?

Reply:
good catch, looks like this isn't needed, and the disabled color wasn't being applied either.